### PR TITLE
feat(L2): gold border & optimism dropdown

### PR DESCRIPTION
**Pull request by JChiaramonte7**
image

### CODE REVIEW

**Comment by vbstreetz** [comment]
- [ ] App crashes after switching networks for some reason https://photos.app.goo.gl/kWVeAqYf6sycUbD9A

**Comment by kmeraz** [comment]
I saw your comment @vbstreetz so I tried switching networks and I didn't get the same results 🤔  no UI error on my part. For the record, I went from Eth Mainnet as well

**Comment by kmeraz** [comment]
I just tried going from L2 back to L1 and got this:
image

Based on the error, I'm guessing there's a conditional hook somewhere

**Comment by kmeraz** [comment]
Dang, maybe Github Actions caught that 💪 . Viewable in the diff:
image

**Comment by kmeraz** [comment]
Looks like the bug is gone 🎉

File: sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx [hunk 1]
```diff
@@ -29,11 +74,74 @@ const NetworksSwitcher: FC<NetworksSwitcherProps> = () => {
...
+					viewBox={`0 0 ${CaretDownIcon.width} ${CaretDownIcon.height}`}
+				/>
+			</components.DropdownIndicator>
+		);
+	};
+
 	return !isL2 ? (
 		<Container onClick={switchToL2}>
-			<Button>{t('header.networks-switcher.l2')}</Button>
+			{!isMainnet && <Button>{t('header.networks-switcher.l2')}</Button>}
```
Comment (kmeraz) [inline sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx:126]: would we rather just not render the container at all? if we're not on mainnet

**Review by kmeraz — APPROVED** [review]
🚀

**Review by dvd-schwrtz — APPROVED** [review]

File: sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx (hunk 1 above)
Comment (JChiaramonte7) [reply sections/shared/Layout/AppLayout/Header/NetworksSwitcher.tsx:126]: yeah that's better but im going to revert this in a week

**Pull request merged**